src/cryptonote_protocol/cryptonote_protocol_defs.h

```diff
@@ -252,6 +252,8 @@ namespace cryptonote
     struct request
     {
       std::array<uint16_t, 3> snode_version;
+      std::array<int, 3> storage_version;
+      std::array<int, 3> lokinet_version;
```

Comment:
These should be `uint16_t` (it's smaller, but also it's a precisely defined type while `int` is looser), and also need to get added in the cryptonote_protocol_defs.cpp